doc/conf.py

```diff
@@ -0,0 +1,293 @@
+sys.path.insert(0, os.path.abspath('./../pynest/nest'))
+
+source_suffix = ['.rst', '.md']
+source_parsers = {
+    '.md': CommonMarkParser
+}
+
+# -- Checking for pandoc --------------------------------------------------
+
+try:
+    print(check_output(['pandoc', '--version']))
+except CalledProcessError:
+    print("No pandoc on %s" % os.environ['PATH'])
+
+
+for dirpath, dirnames, files in os.walk(os.path.dirname(__file__)):
+    for f in files:
+        if f.endswith('.md'):
+            ff = os.path.join(dirpath, f)
+            print(ff)
+            fb = os.path.basename(f)[:-3]
+            print(fb)
+            fo = fb + ".rst"
+            args = ['pandoc', ff, '-o', fo]
+            # check_output(args)
+            # check_output(args)
+
+# -- General configuration ------------------------------------------------
+
+# import errors on libraries that depend on C modules
+# http://blog.rtwilson.com/how-to-make-your-sphinx-documentation-compile-with-readthedocs-when-youre-using-numpy-and-scipy/
+class Mock(MagicMock):
+    @classmethod
+    def __getattr__(cls, name):
+            return MagicMock()
+
+
+MOCK_MODULES = ['numpy', 'scipy', 'matplotlib', 'matplotlib.pyplot', 'pandas']
+sys.modules.update((mod_name, Mock()) for mod_name in MOCK_MODULES)
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+# extensions = [
+#    'sphinx.ext.autodoc',
+#    'sphinx.ext.napoleon',
+#    'sphinx.ext.autosummary',
+#    'sphinx.ext.doctest',
+#    'sphinx.ext.intersphinx',
+#    'sphinx.ext.todo',
+#    'sphinx.ext.coverage',
+#    'sphinx.ext.mathjax',
+#    'sphinx_gallery.gen_gallery',
+# ]
+
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.napoleon',
+    'sphinx.ext.autosummary',
+    'sphinx.ext.doctest',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.todo',
+    'sphinx.ext.coverage',
+    'sphinx.ext.mathjax',
+]
+
+# sphinx_gallery_conf = {
+#    'doc_module': ('sphinx_gallery', 'numpy'),
+#    # path to your examples scripts
+#    'examples_dirs': '../pynest/examples',
+#    # path where to save gallery generated examples
+#    'gallery_dirs': 'auto_examples',
+#    'backreferences_dir': False
+# }
+
+
+
+mathjax_path = \
+    "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/MathJax" \
+                ".js?config=TeX" \
+              "-AMS-MML_HTMLorMML"
+
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+# source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'contents'
+
+# General information about the project.
+project = u'NEST simulator user documentation'
+copyright = u'2017, nest-simulator'
```

Comment:
Why at 2017 copyright? Generally, we put everything to 2004.

Reply:
Copyright changed to 2014.